src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -15,11 +16,9 @@ import FastImage from 'react-native-fast-image';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
 import MemoIcArrowBackCircle from '../../../assets/arrow/Ic_arrow_back_circle';
 import ShareIconCircle from '../../../assets/icons/Ic_share_circle';
-import TopicSignedDefaultIcon from '../../../assets/dp-topics-signed.png';
-import TopicAnonDefaultIcon from '../../../assets/dp-topics-anon.png';
 import {Shimmer} from '../../../components/Shimmer/Shimmer';
 import dimen from '../../../utils/dimen';
-import {normalize} from '../../../utils/fonts';
+import {normalize, normalizeFontSize} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The imports for `TopicSignedDefaultIcon` and `TopicAnonDefaultIcon` have been removed, and `normalizeFontSize` has been added. The removal of these imports is consistent with the changes made in the rendering logic of images where these icons are no longer used. The addition of `normalizeFontSize` is used later in the styles definition.
```diff
- import TopicSignedDefaultIcon from '../../../assets/dp-topics-signed.png';
- import TopicAnonDefaultIcon from '../../../assets/dp-topics-anon.png';
+ import {normalize, normalizeFontSize} from '../../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->